packages/ag-charts-website/src/content/docs/layout/_examples/chart-class/styles.css

```diff
@@ -0,0 +1,3 @@
+.chart {
```

Comment:
I don't think this class is actually referenced by the example. When I run the example locally I don't see this file in the files list, and no red border.

Reply:
I think you have to save this file to force it to rebuild the example

Reply:
Definitely worked locally for me - maybe it's easier to just revisit if it doesn't show up on staging

Reply:
Sorry my bad. Wasn't aware about the save thing. Can confirm all works locally for me. Everything looking great 👍